### PR TITLE
Add option to not handle Last Active on Manual Instance

### PR DESCRIPTION
**Pull request by SkySom**
This should allow opening other versions of the Manual without overwriting.

also includes getter for IMultiblock

### CODE REVIEW

File: src/main/java/blusunrize/lib/manual/gui/ManualScreen.java [hunk 1]
```diff
@@ -154,7 +163,11 @@ else if(currentNode.getSuperNode()!=null)
 		}
 		else if(searchField!=null)
 			searchField = null;
-		lastActiveManual = this;
+
+		if (doLastActive)
+		{
+			lastActiveManual = this;
+		}
```
Comment (malte0811) [inline src/main/java/blusunrize/lib/manual/gui/ManualScreen.java]: Formatting (no baces on single-line `if`s)

File: src/main/java/blusunrize/lib/manual/gui/ManualScreen.java [hunk 2]
```diff
@@ -55,7 +55,14 @@
 	private ClickableList entryList;
 	private ClickableList suggestionList;
 
+	private final boolean doLastActive;
```
Comment (malte0811) [inline src/main/java/blusunrize/lib/manual/gui/ManualScreen.java]: I would have called it something different here (`setAsLastActive` or similar), but this works as well.

**Review by malte0811 — CHANGES_REQUESTED** [review]

File: src/main/java/blusunrize/lib/manual/gui/ManualScreen.java (hunk 2 above)
Comment (BluSunrize) [reply src/main/java/blusunrize/lib/manual/gui/ManualScreen.java]: I agree, naming here is a bit awkward.

**Review by BluSunrize — APPROVED** [review]
LGTM

**Review by malte0811 — APPROVED** [review]

**Pull request merged**